### PR TITLE
Add cloud events handler

### PR DESCRIPTION
**Pull request by toddkazakov**

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -3,7 +3,7 @@
   // Example: https://github.com/jshint/jshint/blob/master/examples/.jshintrc
   // Documentation: http://www.jshint.com/docs/
   "browser": true,
-  "esnext": true,
+  "esversion": 8,
```
Comment (toddkazakov) [inline .jshintrc:6]: According to jshint `async/await` is available only in ES8.

**Review by pazaan — APPROVED** [review]
LGTM! :shipit:

**Review by derrickburns — APPROVED** [review]
I cannot follow callbacks. Async/await is sooo much better. Good job.

One question.  See comments.

File: lib/events.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  deleted: 'users:delete',
+};
+
+async function createEventConsumer(config, userEventsHandler, log) {
+  assertConfigValid(config);
+
+  const opts = {
+    clientId: config.cloudEventsSource,
+    brokers: config.brokers,
+    logLevel: logLevel.ERROR,
```
[the remark this answers is not in the document]
Comment (toddkazakov) [reply lib/events.js]: Good suggestion - fixed.

**Pull request merged**